developer/env/localhost_access.env

```diff
@@ -1,6 +1,12 @@
+# For Server and Executor
+CGO_ENABLED=0
 ARMADA_REDIS_ADDRS="localhost:6379"
 ARMADA_EVENTSAPIREDIS_ADDRS="localhost:6379"
 ARMADA_EVENTAPI_POSTGRES_CONNECTION_HOST="localhost"
 ARMADA_POSTGRES_CONNECTION_HOST="localhost"
 ARMADA_PULSAR_URL="pulsar://localhost:6650"
 ARMADA_APICONNECTION_ARMADAURL="localhost:50051"
+
+# For Executor
```

Comment:
Is this cruft?

Reply:
Explained seperately the reasoning for this: but essentially I wanted somewhere to document potentially useful env vars, without creating a markdown file. I think having them here makes sense.